classes/class-pushover-api.php

```diff
@@ -184,7 +201,7 @@ function send() {
 		$request = array(
 						'token' 	=> $this->site_api,
 						'user'  	=> $this->user_api,
-						'title'     => $this->title,
+						'title'     	=> $this->title,
 						'message'	=> $this->message,
 						'url' 		=> $this->url,
```

Comment:
You need to add

```
'sound'   => $this->sound,
```

in the request array.

Reply:
Thanks, I added it. I wasn't sure because there also wasn't a Priority in this array.